tesseract/server.py

```diff
@@ -30,6 +31,8 @@ def __init__(self, redis_host=None):
         self.redis = redis.StrictRedis(host=redis_host, port=6379, db=0)
         self.redis.set('tesseract_server', 1)
 
+        self.notifications = {}
```

Comment:
Let's not do this now. In the future it can be a table similar to INFORMATION_SCHEMA.